src/nbl/video/CVKSwapchain.cpp

```diff
@@ -0,0 +1,96 @@
+#include "nbl/video/CVKSwapchain.h"
+
+#include "nbl/video/CVKLogicalDevice.h"
+#include "nbl/video/surface/ISurfaceVK.h"
+#include "nbl/video/CVulkanImage.h"
+
+namespace nbl {
+namespace video
+{
+
+CVKSwapchain::CVKSwapchain(SCreationParams&& params, CVKLogicalDevice* dev) : ISwapchain(std::move(params)), m_device(dev)
+{
+    VkSwapchainCreateInfoKHR ci;
+    ci.sType = VK_STRUCTURE_TYPE_SWAPCHAIN_CREATE_INFO_KHR;
+    ci.pNext = nullptr;
+    ci.minImageCount = params.minImageCount;
+    ci.clipped = VK_TRUE;
```

Comment:
what does clipped mean?

Reply:
ok I get it now, pixel ownership test... btw pixel ownership is essentially turned off in OpenGL (because when we render to swapchain we actually render to image, so we should set clipped to false in Vulkan I think).

@achalpandeyy